ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -78,11 +79,13 @@ descriptor:
   carbon: "Nonmetallic chemical element with the atomic number of six (e.g. graphite)."
   chilled: "Cold water (typically 40 - 50F)."
   changeover: "Connecting to different equipment ciecuit (e.g. switching between chiller and hot water)."
+  cie: "CIE light color schema"
   circulation: "Process or component used to circulate fluid through a device or system (typically onboard a boiler)."
   co: "Measures carbon monoxide concentration."
   co2: "Measures carbon dioxide concentration."
   cogeneration: "Associated with a cogeneration process."
   cold: "Associated with cold area or process."
+  color: "Color of light or of a physical medium"
```

Comment:
would propose a definition that doesnt include itself, such as "Physical perceptual property of light deriving from the electromagnetic spectrum."

Reply:
Changed